src/main/scala/uk/gov/nationalarchives/tdr/api/service/ConsignmentService.scala

```diff
@@ -125,7 +123,8 @@ class ConsignmentService(
 
   def updateSeriesIdOfConsignment(updateConsignmentSeriesIdInput: UpdateConsignmentSeriesIdInput): Future[Int] = {
     for {
-      result <- consignmentRepository.updateSeriesIdOfConsignment(updateConsignmentSeriesIdInput)
+      series <- seriesRepository.getSeries(updateConsignmentSeriesIdInput.seriesId)
+      result <- consignmentRepository.updateSeriesIdAndNameOfConsignment(updateConsignmentSeriesIdInput, Some(series.head.name))
```

Comment:
Using `series.head.name` could lead to an exception if an empty `series` list is returned on line 126. Should do a check to make sure the list is not empty.